src/integrations/prefect-dask/prefect_dask/task_runners.py

```diff
@@ -249,80 +275,161 @@ def __eq__(self, other: object) -> bool:
                 and self.client_kwargs == other.client_kwargs
             )
         else:
-            return NotImplemented
+            return False
+
+    def duplicate(self):
+        """
+        Create a new instance of the task runner with the same settings.
+        """
+        return type(self)(
+            address=self.address,
+            cluster_class=self.cluster_class,
+            cluster_kwargs=self.cluster_kwargs,
+            adapt_kwargs=self.adapt_kwargs,
+            client_kwargs=self.client_kwargs,
+        )
 
-    async def submit(
+    def submit(
         self,
-        key: UUID,
-        call: Callable[..., Awaitable[State[R]]],
-    ) -> None:
+        task: Task,
+        parameters: Dict[str, Any],
+        wait_for: Iterable[PrefectFuture],
+        dependencies: Optional[Dict[str, Set[TaskRunInput]]] = None,
+    ) -> PrefectDaskFuture:
```

Comment:
Every future type that we implement for each task runner will have the same interface, right? It's just there will be some special handling based on the underlying system?

Reply:
Yes, Each specific future will have a different implementation for waiting for a future and getting a future result, but the `.wait`, `.result`, and `.state` interfaces will all be the same.